### PR TITLE
week 3 assignment

### PR DESCRIPTION
**Pull request by SonaliBedge**
Introduction to Express

### CODE REVIEW

File: 02-express-tutorial/app.js [hunk 1]
```diff
@@ -1 +1,62 @@
...
+        filteredProducts = filteredProducts.filter((product) => {
+          return product.name.startsWith(search)
+        })
+      }
+      
+      if (limit) {
+        filteredProducts = filteredProducts.slice(0, Number(limit))
+      }
+      
+    filteredProducts = filteredProducts.filter(product => product.price > 20.00)
```
Comment (ethanarns) [inline 02-express-tutorial/app.js]: Very close! You did the filter correctly, but it has to be in an if statement like before in order to have custom values entered (or for it to be active at all).

File: week_3_alt_assignment/app.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+const app = require("./stuffThatYouCanIgnore"); // take my word for it :)
```
Comment (ethanarns) [inline week_3_alt_assignment/app.js:1]: Absolutely incredible work here!

**Review by ethanarns — CHANGES_REQUESTED** [review]
Very well done! Your Week 3 alt assignment was incredible. Only one small issue remained in the original, but I'm positive you'll be able to figure it out!

File: 02-express-tutorial/app.js (hunk 1 above)
Comment (SonaliBedge) [reply 02-express-tutorial/app.js]: I have solve the issue , please check Whenever it is convenient for you. Thank you for your comment

File: week_3_alt_assignment/app.js (hunk 2 above)
Comment (SonaliBedge) [reply week_3_alt_assignment/app.js:1]: Thank you so much for your comment

**Pull request merged**